src/Router.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import Nav from './components/Nav/Nav';
-import Login from './pages/User/Login/Login';
+import ProductList from './pages/ProductList/ProductList';
+import ProductDetail from './pages/ProductDetail/ProductDetail';
 
 const Router = () => {
   return (
     <BrowserRouter>
-      <Nav />
       <Routes>
-        <Route path="login" element={<Login />} />
+        <Route path="/" element={<ProductList />} />
+        <Route path="/ProductDetail" element={<ProductDetail />} />
```

Comment:
일반적으로 라우팅 경로는 kebab-case로 작성합니다.
```suggestion
        <Route path="/product-detail" element={<ProductDetail />} />
```